### PR TITLE
[INTL] Add file header. Fix typos and improve portuguese (pt-PT) translation. Fix accel collisions

### PR DESCRIPTION
**Pull request by zecarlos1957**
Add file header.
Fix typos and improve portuguese (pt-PT) translation.
Fix accel collisions
Remove leading space

### CODE REVIEW

File: dll/cpl/intl/lang/pt-PT.rc [hunk 1]
```diff
@@ -1,12 +1,19 @@
- LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+/*
+ * PROJECT:     ReactOS Regional Options Control Panel
```
Comment (binarymaster) [inline dll/cpl/intl/lang/pt-PT.rc:2]:
```suggestion
 * PROJECT:     ReactOS International Control Panel
```

File: dll/cpl/intl/lang/pt-PT.rc [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2020-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```
Comment (binarymaster) [inline dll/cpl/intl/lang/pt-PT.rc:5]:
```suggestion
 * TRANSLATOR:  Copyright 2019-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```

File: dll/cpl/intl/lang/pt-PT.rc [hunk 3]
```diff
@@ -52,7 +59,7 @@ BEGIN
     GROUPBOX "&Tabelas de conversão de páginas de código", -1, 5, 101, 234, 88
     CONTROL "", IDC_CONV_TABLES, "SysListView32", LVS_REPORT | LVS_SORTASCENDING | LVS_NOCOLUMNHEADER | WS_BORDER | WS_TABSTOP, 14, 114, 217, 70
     GROUPBOX "Predefinições de conta de utilizador", -1, 5, 193, 234, 32
-    AUTOCHECKBOX "Aplicar todas as definições à conta de utilizador actual e ao perfil de utilizador &predefinido", IDC_APPLY_CUR_USER_DEF_PROFILE, 12, 201, 220, 22, BS_MULTILINE | BS_TOP
+    AUTOCHECKBOX "&Aplicar todas as definições à conta de utilizador actual e ao perfil de utilizador &predefinido", IDC_APPLY_CUR_USER_DEF_PROFILE, 12, 201, 220, 22, BS_MULTILINE | BS_TOP
```
Comment (HBelusca) [inline dll/cpl/intl/lang/pt-PT.rc:62]: Note that there was already an accelerator at `predefinido` (last word)

**Review by HBelusca — CHANGES_REQUESTED** [review]

File: dll/cpl/intl/lang/pt-PT.rc (hunk 3 above)
Comment (zecarlos1957) [reply dll/cpl/intl/lang/pt-PT.rc]: Ye

**Review by JoachimHenze — APPROVED** [review]
Approving the **current state** of the PR.

**Pull request merged**